src/types.ts

```diff
@@ -146,3 +146,24 @@ export interface TriggerQueueUpdatePayload {
   id: number
   status: TriggerStatus
 }
+
+export interface User {
```

Comment:
User and Organisation are imported in routes.ts, but not actually used

Reply:
do we need these types ?

Reply:
Cool, thanks. Left over from when they were being used.

230858ab